cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java

```diff
@@ -372,7 +372,7 @@ private void createNamespaces() {
       Throwables.propagate(e);
     }
     LOG.info("Creating and registering {} namespace", Constants.DEFAULT_NAMESPACE);
-    getStore().createNamespace(new NamespaceMeta.Builder().setId(Constants.DEFAULT_NAMESPACE)
+    getStore().createNamespace(new NamespaceMeta.Builder()
```

Comment:
This class got renamed to ``UpgradeTool``, but hopefully ``git`` should do its job.